configuration/config-txt/overclocking.md

```diff
@@ -10,31 +10,49 @@ Overclocking and overvoltage will be disabled at runtime when the SoC reaches 85
 
 | Option | Description |
 | --- | --- |
-| arm_freq | Frequency of the ARM CPU in MHz; default value is `1000` for Pi Zero and Pi Zero W, `700` for Pi 1, `900` for Pi 2, `1200` for Pi 3, `1400` for Pi 3A+/3B+, `1500` for Pi 4B|
-| gpu_freq | Sets `core_freq`, `h264_freq`, `isp_freq`, and `v3d_freq` together; on Pi 1/Pi 2, the default value is `250` for all items, on Pi 3/Pi Zero /Pi Zero W `core_freq` defaults to `400`, while `h264_freq`, `isp_freq`, and `v3d_freq`default to `300`; on Pi 4B the `core_freq`, `h264_freq`, `isp_freq`, and `v3d_freq` default to `500`, `600` is the only other accepted value|
-| core_freq | Frequency of the GPU processor core in MHz, influences CPU performance because it drives the L2 cache and memory bus; default value is `250` for Pi 1/Pi 2 and `400` for Pi 3/Pi Zero/Pi Zero W. For Pi 4B the default value is `500`, `600` is the only other accepted value; the L2 cache benefits only Pi Zero/Pi Zero W/ Pi 1, there is a small benefit for SDRAM on Pi 2/Pi 3 and Pi 4B|
+| arm_freq | Frequency of the ARM CPU in MHz.
+| gpu_freq | Sets `core_freq`, `h264_freq`, `isp_freq`, and `v3d_freq` together; on Pi 3/Pi Zero /Pi Zero W `core_freq` defaults to `400`, while `h264_freq`, `isp_freq`, and `v3d_freq`default to `300`; on Pi 4B this can be set to 500 or 600 only|
```

Comment:
Is the `gpu_freq` / `core_freq` on the Pi4 only settable to 600 when neither `hdmi_enable_4kp60` or `enable_tvout` are being used?

Reply:
Not sure.

Reply:
@popcornmix ?

Reply:
On Pi4 core is fixed, based on display in use (500MHz default, 550MHz when hdmi_enable_4kp60=1 and 360MHz when enable_tvout=1). Anything else is unsupported. Note: these aren't set in stone, and are subject to change if we find combinations of plls/clocks that give better performance/power/stability.

The other gpu clocks can be changed with the same rules as on Pi0-3 (i.e. we try to do what you ask for but no overclocks are guaranteed).

Reply:
Oh, so does that mean that the table elsewhere on this page that says that `enable_tvout` sets a 432 MHz core clock on the Pi4B is wrong? (and it should in fact be 360MHz?)

Does this also mean that any `core_freq` setting on the Pi4 is effectively ignored, and setting `gpu_freq` on the Pi4B sets `h264_freq`, `isp_freq`, and `v3d_freq` together, but _doesn't_ set `core_freq`?

Reply:
Yes, 432MHz is incorrect (it may have been correct before DVFS firmware changes some months ago). core_freq is largely ignored.

PLLC is set based on hdmi_enable_4kp60/enable_tvout. Currently that is:
3000 default
3300 hdmi_enable_4kp60
3240 enable_tvout

There is then a fixed divide by 3 for the channel frequency:
1000 default
1100 hdmi_enable_4kp60
1140 enable_tvout

We then pick the integer divider that doesn't exceed core_freq where:
core_freq=550 hdmi_enable_4kp60
core_freq=500 otherwise

You can specify other core_freq values that are integer divisors of the channel frequency.
There are very few options that work. core_freq=1000 will never boot. Any others are underclocks (which are supported - e.g. core_freq=333 for default display case).
Technically enable_tvout=1 core_freq=570 (with a suitable over_voltage) should be possible but that is a very niche case.

But these are all implementation details that will likely change during the lifetime of Pi4 and as such I don't think they are worth documenting in detail. Currently there is almost nothing that can usefully be done by a user to overclock core freq. But that could change.

Reply:
I've updated to fix up the issues Dom has pointed out.